packages/office-ui-fabric-react/src/components/Button/BaseButton.styles.ts

```diff
@@ -51,7 +50,6 @@ export const getStyles = memoizeFunction(
           textDecoration: 'none',
           textAlign: 'center',
           cursor: 'pointer',
-          verticalAlign: 'top',
```

Comment:
hmm we have `display: 'inline-block',` set above.

Reply:
but i agree this should be removed :)

Reply:
Screener did not report any failures. Thus I suspect there is no delta in pixels painted by the browser with this attribute as it reported. If there is a regression, please escalate to me. Thanks.